packages/common/test/chains.spec.ts

```diff
@@ -23,6 +24,33 @@ describe('[Common/Chains]: Initialization / Chain params', () => {
     )
   })
 
+  it('Deep copied common object should have parameters that are independent of the original copy', async () => {
+    let chainConfig: ChainConfig
+    let c: Common
+    const setCommon = async () => {
+      chainConfig = Mainnet
```

Comment:
Haha, so funny, I am pretty sure (or: could very much imagine) that this is where you turned wrong: so, by associating `chainConfig = Mainnet` here and *then* manipulating this further down the line with `      chainConfig.chainId = 2` this then also manipulates `Mainnet` itself.

And *that* causes another test below in line 55 to fail. 😂 And the error looks pretty much like a "still failing" message if one is not looking pretty closely. 🙂